ci/tests/sonarqubetest/sonar_auth_test.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+    "os"
+    "net/http"
+    "github.com/DATA-DOG/godog"
+    "io/ioutil"
+    "fmt"
+    "net/url"
+    "time"
+    "strings"
+)
+
+var sonarUrl string
+var body string
+var httpClient = &http.Client{
+      Timeout: time.Second * 10,
+}
+
+const (
+    SONAR_AUTH_ENDPOINT            string = "/api/authentication/login"
+    SONAR_GITHUB_AUTH_ENDPOINT     string = "/sessions/new?return_to=%2Fprojects"
+    SONAR_TEST_STANDALONE_USER     string = "teststandaloneuser"
+    SONAR_TEST_STANDALONE_PASSWORD string = "TEST_STANDALONE_PASSWORD"
+    SONAR_FAIL_TEST_PASSWORD       string = "iamafailureinlogins"
+)
+
+func getSonarEndPoint(path string) string {
+    sonarUrl = os.Getenv("SONARQUBE_HOST")
```

Comment:
Not sure where this env var is defined, but it would be good to call it SONAR_URL. Jenkins might still use JENKINS_HOST, but we've corrected such usage elsewhere.

Reply:
Hi @danieljamesrees the env var is defined in the followng yaml file 
https://github.com/FINkit/sonarqube-boshrelease/blob/master/ci/tasks/test-sonarqube.yml

Reply:
@danieljamesrees change SONARQUBE_HOST to SONAR_URL as requested :)